app/models/label/LabelTable.scala

```diff
@@ -375,10 +376,34 @@ object LabelTable {
       "label_type_value" -> labelMetadata.labelTypeValue,
       "severity" -> labelMetadata.severity,
       "temporary" -> labelMetadata.temporary,
+      //this one is different because we're pulling from the tags and label_tags databases
+      "tags" -> getTagsFromLabelId(labelMetadata.labelId),
       "description" -> labelMetadata.description
     )
   }
 
+  //seperate sql request for getting tags, gets all the tags that correspond to the label Id.
+  def getTagsFromLabelId(labelId: Int): String = db.withSession { implicit session =>
+      val getTagsQuery = Q.query[Int, (String)](
+        """SELECT tag FROM sidewalk.tag
+             WHERE tag.tag_id IN (
+                  SELECT tag_id FROM sidewalk.label_tag
+                        WHERE label_tag.label_id = ?
+             )""".stripMargin
+      )
+      var tags = getTagsQuery(labelId).list
+      var str = ""
+      var isFirst = true
+      for(tag <- tags){
+        if(!isFirst){
+          str += ", "
+        }
+        isFirst = false;
+        str += tag
+      }
+      return str
```

Comment:
You should check out the mkString function! I believe that the 11 lines starting with `var tags = ...` and ending with `return str` could be replaced with just `getTagsQuery(labelId).list.mkString(", ")`

Some other notes for the future, even though you will probably remove the code:
* For any variable that is never modified after it is created, you can should `val` instead of `var`, which means that it cannot ever be modified (it is immutable). When writing in Scala, you want to almost never use vars, using vals almost always. One way to help with that is that you almost never want to loop through a list. If you ever want to start looping through a list, check out the `map` function and use that instead :)
* You don't have to use the `return` keyword in Scala, you can just end with the object you are returning!